examples/pytorch/pinsage/model.py

```diff
@@ -50,19 +52,19 @@ def train(dataset, args):
     g.nodes[item_ntype].data['id'] = torch.arange(g.number_of_nodes(item_ntype))
 
     # Prepare torchtext dataset and vocabulary
-    fields = {}
-    examples = []
-    for key, texts in item_texts.items():
-        fields[key] = torchtext.legacy.data.Field(include_lengths=True, lower=True, batch_first=True)
+    textset = {}
+    tokenizer = get_tokenizer(None)
+
+    textlist = []
+    batch_first = True
+
     for i in range(g.number_of_nodes(item_ntype)):
-        example = torchtext.legacy.data.Example.fromlist(
-            [item_texts[key][i] for key in item_texts.keys()],
-            [(key, fields[key]) for key in item_texts.keys()])
-        examples.append(example)
-    textset = torchtext.legacy.data.Dataset(examples, fields)
-    for key, field in fields.items():
-        field.build_vocab(getattr(textset, key))
-        #field.build_vocab(getattr(textset, key), vectors='fasttext.simple.300d')
```

Comment:
> Let's simply remove L41-43.

Here, as L65 has been commented out and build_vocab() seems never uses vectors. I think it should be okay to remove BagOfWordsPretrained?

Reply:
Sounds good to me. Let's also re-invoke the experiments for a sanity check after removing that.

Reply:
Sure. I have already checked for both scenarios (model.py and model_sparse.py), and they produced consistent results after completely removing BagOfWordsPretrained class.  I think the script author originally used such class only when vocab embedding vectors are created using "fasttext.simple.300d" (e.g. L65 in model.py). But I can never find this file in current version.